fe/fe-core/src/main/java/com/starrocks/backup/RestoreJob.java

```diff
@@ -795,6 +796,9 @@ private void checkAndPrepareMeta() {
             return;
         }
 
+        // add all restored functions
+        addRestoredFunctions(db);
```

Comment:
should restore functions after restore finished ? which does not need to rollback when restore failed.
also need to check the restore for view.

Reply:
Updated, move to the very last place, but i think view should be unchanged because it is use the same logical to be restore as the normal table.